subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/Checkstyle.java

```diff
@@ -364,19 +364,7 @@ public void setIgnoreFailures(boolean ignoreFailures) {
      * @since 3.4
```

Comment:
🤔 The `@return` tag does not make sense anymore. I guess for `Property` properties we should just remove the `@return` tags?